test/Microsoft.Framework.Internal.Test/BufferedHtmlContentTest.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+#if DNXCORE50
+using System.Reflection;
+#endif
+using Microsoft.AspNet.Html.Abstractions;
+using Microsoft.Framework.WebEncoders;
+using Microsoft.Framework.WebEncoders.Testing;
+using Xunit;
+
+namespace Microsoft.Framework.Internal
+{
+    public class BufferedHtmlContentTest
```

Comment:
Surprised this class isn't enough like `BufferEntryCollectionTest` that GitHub considers it a rename. Is this all because `BufferedHtmlContent` doesn't implement `IEnumerable<string>`?